src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.Initialization.cs

```diff
@@ -49,6 +49,6 @@ internal static void Initialize()
         }
 
         [LibraryImport(Libraries.AndroidCryptoNative, EntryPoint = "CryptoNative_EnsureOpenSslInitialized")]
-        private static partial int EnsureOpenSslInitialized();
+        private static partial int EnsureOpenSslInitialized(void* mallocFunction, void* reallocFunction, void* freeFunction);
```

Comment:
This p/invoke signature no longer matches Android's function. I don't think it will lead to stack red zone or shadow space problems, but might as well make sure the parameters match.

https://github.com/dotnet/runtime/blob/aab8803eac4111a711b6eb65242ee8a5b14fa411/src/native/libs/System.Security.Cryptography.Native.Android/pal_misc.c#L7